brainstormforce/astra
# AST-1919 - Table block not aligned centrally for Legacy block editor setup.

### Description
 Table block not aligned centrally for Legacy block editor setup.

### Screenshots
Screenshots:
https://share.bsf.io/12uLJO21
https://share.bsf.io/Qwun0epl

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Create a table block (Make sure the legacy option is set in the customizer)
- Center align it
- Check on front-end

### Checklist:
